recipes/breakfest/arabic_omelette.md

```diff
@@ -0,0 +1,35 @@
+# Arabic Omellete
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-10%25-brightgreen.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-10min-brightgreen.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Very%20Positive-brightgreen.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-Very%20Cheap-brightgreen.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-False-red.svg)
+
++ **First cooked by:**
+[@Karyum](https://github.com/karyum)
+
++ **Date:** Week 5 FACN2
++ **Author of Recipe:**
+#MrsSaliba
+
+### Ingredients
+- 2 small eggs.
+- a handful of parsley.
+- a handful of green/spring onions.
+- zaatar to taste.
+- salt and pepper
+
+### Preparation
+
+1. Chop the parsley and the green/spring onions into small pieces.
+2. add the everything into a small bowl and stir it with a **fork**.
+3. heat up a pan with a bit olive oil and wait for 1 min untill it heats up.
+4. add all of the mix into the heated pan.
+5. from here it's self preference on how much you want the Omellete to be cooked, I would say flip it when gets golden-red.
```

Comment:
I would change the wording of this slightly:
> it is up to you how much you want your omelette to be cooked but I would recommend flipping when its golden-red

Reply:
is golden red even a colour? :upside_down_face:

Reply:
I'm sorry are you color blind ? do you even color mate ? have you seen the color gold ? have you seen the color red ?
connect the dotes

Reply:
tbf golden brown is the more common phrase - and also more appropriate in this case maybe?

Reply:
>I would change the wording of this slightly:

I think this comment isn't entirely useful without actually suggesting how or why mate 😕

Reply:
good point mate i trust you @m4v15 bless you

Reply:
@m4v15 I agree but that's me using bad markdown! That quote in my review comment is what I think it should be updated to :+1: open to other suggestions though

Reply:
@matthewdking oh I see, yeah I thought you were just quoting it

Reply:
@Karyum since when you trust @m4v15 on Naz recipes??